__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -191,229 +190,216 @@ describe('TaskDetails Page', () => {
         const errorElement = await screen.findByText('Something went wrong!');
         expect(errorElement).toBeInTheDocument();
     });
+
     it('Renders Task Started-on Date', async () => {
-        const { getByText } = renderWithRouter(
+        const { queryAllByText } = renderWithRouter(
             <Provider store={store()}>
                 <Details
                     detailType={'StartedOn'}
-                    value={'3/30/2021, 12:00:00 AM'}
+                    value={'3/30/2024, 11:20:00 AM'}
                 />
             </Provider>
         );
         await waitFor(() => {
-            expect(getByText('3/30/2021, 12:00:00 AM')).toBeInTheDocument();
+            const dateElements = queryAllByText('3/30/2024, 11:20:00 AM');
+            expect(dateElements.length).toBeGreaterThan(0);
```

Comment:
this will be used in ends on and started on right? so there will be 2 appearances so it can be toBe(2)

Reply:
if I'm applying your suggestion test is failing `expect(dateElements.length).toBe(2)`;

Reply:
I understand that having two appearances of the text `3/30/2024, 11:20:00 AM` might be expected in scenarios like `Ends On` and `Started On`. However, upon closer examination of the component's logic and the context in which it's used, it seems that the current implementation only renders one instance of this text.

Reply:
if I'm changing this `toBe(2)` test is failing.